src/pages/Filing/UpdateFinancialProfile/UpdateIdentifyingInformation.tsx

```diff
@@ -73,11 +73,10 @@ function UpdateIdentifyingInformation({
     <FormSectionWrapper>
       <SectionIntro heading='Update your financial institution identifying information'>
         If your financial institution does not have an RSSD ID, provide your TIN
-        below. If your financial institution has an RSSD ID number and “Not
-        applicable” is shown below, contact your primary federal regulator,
-        state regulator, or reserve bank to link your LEI to your RSSD ID in
-        NIC. Once updated in NIC, we will populate the data in our system
-        accordingly.
+        below. If your institution has an RSSD ID but quot;Not applicablequot;
```

Comment:
```suggestion
        below. If your institution has an RSSD ID but &quot;Not applicable&quot;
```

Missed this here:
<img width="801" alt="Screenshot 2024-10-22 at 4 56 46 PM" src="https://github.com/user-attachments/assets/9b6ac9d7-f479-417c-953f-19bd2b04e308">